### ISSUE TITLE
Curious browser error when using client or provider

### ISSUE THREAD

**Issue by jessebmiller** [Web3-API/prototype#473]
# Situation

I'm starting a new vite, react, typescript project and when I try to use either the polywrap client or provider I get the following error in the browser console

```
json3.js:38 Uncaught TypeError: Cannot read property 'prototype' of undefined
    at runInContext (json3.js:38)
    at json3.js:872
    at ../node_modules/json3/lib/json3.js (json3.js:894)
    at __require2 (chunk-A5AMJUWA.js?v=6235435b:15)
    at ../node_modules/util-inspect/index.js (index.js:12)
    at __require2 (chunk-A5AMJUWA.js?v=6235435b:15)
    at ../node_modules/@web3api/client-js/node_modules/@web3api/tracing-js/build/index.js (index.ts:11)
    at __require2 (chunk-A5AMJUWA.js?v=6235435b:15)
    at ../node_modules/@web3api/client-js/node_modules/@web3api/core-js/build/types/Query.js (Query.ts:3)
    at __require2 (chunk-A5AMJUWA.js?v=6235435b:15)
 ```

# Steps to reproduce

Start a new vite project, install client-js and react (I'm on prealpha-42)

```
$ yarn create vite repro --template react-ts
$ cd repro
$ yarn add @web3api/react
$ echo "v14.16.0" > .nvmrc
$ nvm use
```

Then import and use them. Edit `src/App.tsx` replacing the contents with...

```
import React from 'react';
import { Web3ApiProvider } from '@web3api/react';

export default function App() {
  return (
    <Web3ApiProvider>
      <div>{'Polywrap enabled dApp goes here!'}</div>
    </Web3ApiProvider>
  );
}
```

I'm only showing the error with `Web3ApiProvider` but I've experienced the same thing when trying to use the client without the provider.

Run the dev server

`$ yarn dev`

check `localhost:3000` and the error should show up in the console.

# Notes

Here is the `package.json` I ended up with after that

```
{
  "name": "repro",
  "version": "0.0.0",
  "scripts": {
    "dev": "vite",
    "build": "tsc && vite build",
    "serve": "vite preview"
  },
  "dependencies": {
    "@web3api/react": "^0.0.1-prealpha.42",
    "react": "^17.0.0",
    "react-dom": "^17.0.0"
  },
  "devDependencies": {
    "@types/react": "^17.0.0",
    "@types/react-dom": "^17.0.0",
    "@vitejs/plugin-react-refresh": "^1.3.1",
    "typescript": "^4.3.2",
    "vite": "^2.5.4"
  }
}
```

**Comment by dOrgJelli** [comment]
Hey @jessebmiller, thanks so much for making this issue.

I found the issue. It is with the `util-inspect` package we are using in the `@web3api/tracing-js` package. It includes a dependency `json3` which does not have proper browser support.

I'll be pushing a new release shortly with the fix.

**Comment by dOrgJelli** [comment]
Hey @jessebmiller, thanks so much for making this issue.

I found the issue. It is with the `util-inspect` package we are using in the `@web3api/tracing-js` package. It includes a dependency `json3` which does not have proper browser support.

I'll be pushing a new release shortly with the fix.

**Comment by dOrgJelli** [comment]
Hey @jessebmiller, thanks so much for making this issue.

I found the issue. It is with the `util-inspect` package we are using in the `@web3api/tracing-js` package. It includes a dependency `json3` which does not have proper browser support.

I'll be pushing a new release shortly with the fix.

**Comment by dOrgJelli** [comment]
This is fixed in version `0.0.1-prealpha.43`
